packages/Takeover/src/components/Header/Header.js

```diff
@@ -38,14 +41,17 @@ const propTypes = {
   level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
   onClose: PropTypes.func,
   refHeading: RefOf(),
+  /** Add node object to the right side of heading next to the close button */
+  headerTools: PropTypes.node,
```

Comment:
This component is already called `Header`, so maybe the prop can be just `tools`? or maybe just `body` ?

Reply:
we do have `body`  let me change it to `tools`